source/constraints.py

```diff
@@ -244,7 +244,7 @@ def get_reduced_constraints(idx_a, idx_f, anchors, frame):
                           axis=1)
 
 
-def get_frame(n_constraints, n_positions):
+def get_frame(n_constraints, n_positions):  # TODO(michalina) allow random times
```

Comment:
Is this here on purpose?

Reply:
Ups, sorry :D I mean, I should add random times anyway, but I shouldn't announce it here ;)